action.yaml

```diff
@@ -12,6 +12,10 @@ inputs:
     description: "Path to the modules directory"
     required: false
     default: "scanners/"
+  docs_url:
+    description: "URL to the documentation"
+    required: false
+    default: "https://docs.boostsecurity.net"
```

Comment:
Since there's a default here, you probably don't need to `getenv` as it will always be present no?